index.html

```diff
@@ -0,0 +1,225 @@
+            >
+              Forum
+            </h3>
+            <div class="col p-0 text-white">
+              Have the time to share your thoughts and opinions with expets for
+              each topic.
+            </div>
+          </div>
+          <div class="cards row align-items-center gap-1 fs-6 p-1 mt-3">
+            <div class="col-2 p-0 d-flex justify-content-center">
+              <img src="img/program-icon-4.png" alt="Workshop icon" />
+            </div>
+            <h3
+              class="col-4 fw-bold p-0 text-center fs-5 p-md-0 d-flex justify-content-center"
+            >
+              Workshop
+            </h3>
+            <div class="col p-0 text-white">
+              Try creating your own work using open source license rather than
+              just watching.
+            </div>
+          </div>
+          <div class="cards row align-items-center gap-1 fs-6 p-1 mt-3">
+            <div
+              class="col-2 p-0 d-flex justify-content-center d-flex justify-content-center"
+            >
+              <img src="img/program-icon-5.png" alt="Ignite icon" />
+            </div>
+            <h3 class="col-4 fw-bold p-0 text-center fs-5 p-md-0">AI Ignite</h3>
+            <div class="col p-0 text-white">
+              Get opportunities to network with AI affiliates around the world,
+              also after the Summit.
+            </div>
+          </div>
+        </div>
+        <button class="hidden-mobile main-button text-white fs-5 btn p-2 m-4">
+          Join AI-B Global Summit 2023
+        </button>
+        <a
+          href="#"
+          class="hidden-mobile hidden-desktop text-white text-decoration-underline text-uppercase"
+          >See the whole program</a
+        >
+      </section>
+      <!-- Featured Speakers Section -->
+      <section id="speakers"></section>
+      <!-- Partners section -->
+      <section class="partners-section p-3 p-lg-5 h-100">
+        <h2 class="text-center text-white">Partners</h2>
+        <div class="guide-bar"></div>
+        <ul
+          class="partners d-flex gap-3 flex-wrap fs-4 fw-bold p-3 justify-content-center"
+        >
+          <li class="">OpenAI</li>
+          <li class="">Google</li>
+          <li class="">Microsoft</li>
+          <li class="">IBM</li>
+          <li class="">NVIDIA</li>
+        </ul>
+      </section>
+    </main>
+    <!-- Footer section -->
+    <footer>
+      <section
+        class="footer-section d-flex gap-5 p-4 fs-6 justify-content-center align-items-center bg-white"
+      >
+        <a href="index.html"
+          ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+          <span>B</span>rains</a
+        >
+        <div class="d-flex flex-column text-center">
+          <small class="fw-bold">2023 AIBrains Corp.</small>
+          <small class="fw-bold">All rights reserved</small>
+        </div>
+      </section>
+    </footer>
```

Comment:
- [ ] Great job for adding the footer but in the Mobile version in Home page if you check the given design there is no partner section and footer so I recommend to remove them from the home page in the mobile version 💯🔥🚀✨

- Your design
![image](https://user-images.githubusercontent.com/58553711/235037921-953f1b22-f4fd-4dcf-b80e-bb086698e6ca.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/235008775-77c59901-95d4-4e8c-b4c3-c59b7c9bd200.png)